webcurator-webapp/src/main/resources/application-local+mysql.properties

```diff
@@ -2,11 +2,11 @@
 
 ## Database properties
 databaseType=mysql
-schema.name=DB_WCT
-schema.url=jdbc:mysql://localhost:3306/DB_WCT
+schema.name=DB_WCT_V3
```

Comment:
Won't this break existing installations that rely on these defaults?

Reply:
Sorry.  This is my local configuration.  I would update it. Very grateful for reminding me.

Reply:
Hi, @hannakoppelaar , I've updated the branch v3.0/fix-prod-issues to change the configuration to the default values.